grid2op/tests/test_l2rpn_idf_2023.py

```diff
@@ -8,9 +8,13 @@
 
 
 import grid2op
-from grid2op.gym_compat import GymEnv, BoxGymActSpace, BoxGymObsSpace, DiscreteActSpace
+from grid2op.gym_compat import GymEnv, BoxGymActSpace, BoxGymObsSpace, DiscreteActSpace, MultiDiscreteActSpace
+from grid2op.l2rpn_utils import ActionIDF2023, ObservationIDF2023
+from grid2op.Opponent import GeometricOpponentMultiArea
 import unittest
 import warnings
+import numpy as np
+
 import pdb
 
 class L2RPNIDF2023Tester(unittest.TestCase):
```

Comment:
Add a test that the "obs.was_alert_used_after_attack" is correctly implemented !

Reply:
on it